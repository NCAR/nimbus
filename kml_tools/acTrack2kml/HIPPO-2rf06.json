{"timestamp":"2009-11-12 05:08:45","alt":"336.3","lat":"-43.490498","head":"41.8","declination":"0","lon":"172.529541"}
