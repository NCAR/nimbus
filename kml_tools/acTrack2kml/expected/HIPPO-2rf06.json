{"timestamp":"2009-11-12 05:08:45","alt":"344.2","lat":"-43.490429","head":"41.7","declination":"0","lon":"172.530533"}
